### DISCUSSION TITLE
Can I get GitHub student developer pack without proper school ID/documents?

### DISCUSSION THREAD

**Discussion by donalddongdk** [community/community#19200] [GitHub Education] [answered]
I have a school issued email that's accepted by GitHub and a list of classes I've taken there but concerned it won't be enough to get my application for the pack accepted.

**Comment by justin-2028** [comment, marked as answer]
If you have a school-issued email that's accepted, GitHub will accept your application within an hour as long as your classes list clearly displays your school's name and has dates to accompany the details. Try applying first, as you can always reapply again.

Check this link for more elaboration:
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack

good luck!

**Reply by donalddongdk** [reply]
got it, do you know how many times you can reapply?

**Reply by justin-2028** [reply]
no, but someone tried 3 times and can still reapply
